drivers/include/net/netdev.h

```diff
@@ -245,7 +245,7 @@ typedef enum {
  * May be different for certain radios.
  */
 struct netdev_radio_rx_info {
-    uint8_t rssi;       /**< RSSI of a received packet */
+    int16_t rssi;       /**< RSSI of a received packet in dBm */
```

Comment:
is it really necessary to increase it that much, as far as I understand value can be negative, but typically would range from -100 to +100 or something like that. While `uint8_t` is clearly wrong for all possible RSSI values, IMHO `int8_t` would suffice.

Reply:
This was discussed in #6887. The arguments there against `int8_t` is that LoRa(-like) receivers can work with values below -128dBm.

Reply:
understood, then it makes sense to allow for a larger value range here.